lib/capistrano/tasks/capistrano2.rb

```diff
@@ -34,11 +34,7 @@
     def for_each_process(sidekiq_role, &block)
       sidekiq_processes = fetch(:"#{ sidekiq_role }_processes") rescue 1
       sidekiq_processes.times do |idx|
-        if idx.zero? && sidekiq_processes <= 1
-          pid_file = fetch(:sidekiq_pid)
-        else
-          pid_file = fetch(:sidekiq_pid).gsub(/\.pid$/, "-#{idx}.pid")
-        end
+        pid_file = fetch(:sidekiq_pid).gsub(/\.pid$/, "-#{idx}.pid")
```

Comment:
Um, guys?  

This overrides our pidfile setting in sidekiq.yml, and that's been causing some rather serious problems.